index.html

```diff
@@ -20,73 +21,87 @@
+                <p>Your brain gets <span data-desc="Adjective"></span> but your <span data-desc="Noun"></span> gets dumb</p>
+                <p>So much to <span data-desc="Verb"></span>, so much to see</p>
+                <p>So what's wrong with taking the <span data-desc="Noun"></span> streets?</p>
+                <p>You'll never <span data-desc="Verb"></span> if you don't go</p>
+                <p>You'll never <span data-desc="Verb"></span> if you don't glow</p>
             </div>
             <div class="story toggle hidden" id="story-2">
-                <p>Somebody once asked could I spare some <span>Noun</span> for gas?</p>
-                <p>I need to get myself away from this <span>Noun</span></p>
-                <p>I said, "<span>Expression</span>" what a concept</p>
-                <p>I could use a little <span>Noun</span> myself</p>
-                <p>And we could all use a little change</p>
+                <p>Somebody once asked could I spare some <span data-desc="Noun"></span> for gas?</p>
+                <p>I need to get myself away from this <span data-desc="Noun"></span></p>
+                <p>I said, "<span data-desc="Expression"></span>" what a concept</p>
+                <p>I could use a little <span data-desc="Noun"></span> myself</p>
+                <p>And we could all use a <span data-desc="Adjective"></span>> change</p>
                 <br />
-                <p>Well the <span>Plural Noun</span> start coming and they don't stop coming</p>
-                <p>Fed to the rules and I hit the ground <span>Verb Ending In -ing</span></p>
-                <p>Didn't make sense not to <span>Verb</span> for fun</p>
-                <p>Your brain gets <span>Adjective</span> but your <span>Noun</span> gets dumb</p>
-                <p>So much to <span>Verb</span>, so much to see</p>
-                <p>So what's wrong with taking the <span>Noun</span> streets?</p>
-                <p>You'll never <span>Verb</span> if you don't go</p>
-                <p>You'll never <span>Verb</span> if you don't glow</p>
+                <p>Well the <span data-desc="Plural Noun"></span> start coming and they don't stop coming</p>
+                <p>Fed to the rules and I hit the ground <span data-desc="Verb Ending In -ing"></span></p>
+                <p>Didn't make sense not to <span data-desc="Verb"></span> for fun</p>
+                <p>Your brain gets <span data-desc="Adjective"></span> but your <span data-desc="Noun"></span> gets dumb</p>
+                <p>So much to <span data-desc="Verb"></span>, so much to see</p>
+                <p>So what's wrong with taking the <span data-desc="Noun"></span> streets?</p>
+                <p>You'll never <span data-desc="Verb"></span> if you don't go</p>
+                <p>You'll never <span data-desc="Verb"></span> if you don't glow</p>
             </div>
+            <!--PS: you can easily add more stories by creating another div with the appropriate classes and id-->
```

Comment:
If you have time, I would clean up some of your commented out logic that you no longer need, especially if you feel that you are done with your project and know you no longer will need it.